requirements.txt

```diff
@@ -1,8 +1,8 @@
-django-organizations~=2.0.1
-django-extensions~=3.1.0
-django-allauth~=0.51.0
-django-phonenumber-field~=6.0.0
+django-organizations~=2.1.0
+django-extensions~=3.2.2
+django-allauth~=0.54.0
+django-phonenumber-field~=6.1.0
 phonenumbers>=8.11.0,<8.13.0
```

Comment:
@nemesisdesign is there a reason for not using phonenumbers~=8.13.0 here?

Reply:
I do not remember exactly but I think we can switch to `phonenumbers~=8.13.0` now.